scenarios/sample-download.xml

```diff
@@ -32,8 +32,8 @@
                 <class>com.sonatype.nexus.perftest.maven.DownloadOperation</class>
                 <repo>public</repo>
                 <paths>
-                  <class>com.sonatype.nexus.perftest.maven.HttpdLogParser</class>
-                  <logfile>data/maven-3.1-build-artifact-access.log.gz</logfile>
+                  <class>com.sonatype.nexus.perftest.maven.CsvLogParser</class>
+                  <logfile>data/maven-3.3.4-build-artifact-access.csv.gz</logfile>
```

Comment:
just wondering where the 3.3.4 version comes from - is it related to the upcoming maven release?